engines/mads/game.cpp

```diff
@@ -473,7 +473,8 @@ void Game::synchronize(Common::Serializer &s, bool phase1) {
 		_screenObjects.synchronize(s);
 	} else {
 		// Load scene specific data for the loaded scene
-		_scene._sceneLogic->synchronize(s);
+		if (_scene._sceneLogic)
+			_scene._sceneLogic->synchronize(s);
```

Comment:
So what's the use case here? Is the autosave code trying to create a new saved game? If yes, it would be better to add this check in `MADSEngine::canSaveGameStateCurrently()`, and disallow saving during the intro and any other cutscene